### PR TITLE
[one-cmds] Use default dtype/granularity in qconfig

### PR DESCRIPTION
**Pull request by jinevening**
This uses default dtype/granularity in qconfig file.

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

### CODE REVIEW

**Comment by jinevening** [comment]
This PR enables to omit `quantized_dtype` and `granularity` option when using `one-quantize`, if qconfig file has `default_quantization_dtype` and `default_granularity`.

File: compiler/one-cmds/one-quantize [hunk 1]
```diff
@@ -218,9 +219,29 @@ def _set_default_values(args):
...
-        setattr(args, 'quantized_dtype', 'uint8')
+        if _utils._is_valid_attr(args, 'quant_config'):
+            # Get quantized_dtype from qconfig file
+            try:
+                with open(getattr(args, 'quant_config')) as f:
+                    qconf = json.load(f)
+                    if 'default_quantization_dtype' in qconf:
+                        setattr(args, 'quantized_dtype', qconf['default_quantization_dtype'])
+            except json.decoder.JSONDecodeError:
+                print('Failed to decode ' + getattr(args, 'quant_config') + '. Please check it is a json file.')
```
Comment (jinevening) [inline compiler/one-cmds/one-quantize:230]: This exception is tested by one-quantize_neg_020.test

File: compiler/one-cmds/tests/one-quantize_013.test [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+rm -rf ${outputfile}
+
+# run test without input data
+# quantized_dtype and granularity are not given here
+one-quantize \
+--input_dtype float32 \
+--quant_config one-quantize_013.qconf.json \
+--input_path ${inputfile} \
+--output_path ${outputfile} > /dev/null 2>&1
```
Comment (jinevening) [inline compiler/one-cmds/tests/one-quantize_013.test:42]: This is what this PR enables.

File: compiler/one-cmds/one-quantize (hunk 1 above)
Comment (seanshpark) [reply compiler/one-cmds/one-quantize:230]: what if json file is valid but `default_quantization_dtype` doesn't existing in json ?

Comment (jinevening) [reply compiler/one-cmds/one-quantize:230]: Then, it will use default quantization_dtype, i.e., uint8, as before.

Comment (seanshpark) [reply compiler/one-cmds/one-quantize:230]: But `setattr(args, 'quantized_dtype', 'uint8')` is not called... or is there somewhere else?

Comment (jinevening) [reply compiler/one-cmds/one-quantize:230]: It is called if both quantized_dtype and quant_config are not given. The code is right below the comment (line 232) :)

Comment (seanshpark) [reply compiler/one-cmds/one-quantize:230]: I think if something like this case
```json
{
    "default_quantization_dtype" : "uint8",
    "layers" : [
        {
            "names" : ["InceptionV3..., ..."],
            "dtype" : "int16",
            "granularity" : "channel"
        }
    ]
}
```
or
```json
{
    "default_quantization_dtype" : "uint8",
    "default_granularities" : "channel",
    "layers" : [
        {
            "names" : ["InceptionV3..., ..."],
            "dtype" : "int16",
            "granularity" : "channel"
        }
    ]
}
```

Comment (jinevening) [reply compiler/one-cmds/one-quantize:232]: Ah.. Sorry, I misunderstood the comment. I've updated the PR. Thanks!

**Comment by jinevening** [comment]
@seanshpark  I simplified the code as you suggested :) PTAL

**Review by mhs4670go — APPROVED** [review]
LGTM

**Review by seanshpark — APPROVED** [review]
LGTM thank you!

**Pull request merged**